packages/tc-ui/src/primitives/relationship/server.jsx

```diff
@@ -54,6 +54,11 @@ module.exports = {
 		Object.entries(typeDef.properties)
 			.filter(([, { useInSummary }]) => useInSummary)
 			.forEach(([name]) => props.add(name));
-		return `${propName}_rel {${type} {${[...props].join(' ')}}}`;
+		const relationshipProps = [...new Set(Object.keys(properties))].join(
+			' ',
+		);
+		return `${propName}_rel {${type} {${[...props].join(
```

Comment:
assign `[...props].join(' ')` to a variable on the previous line so that we avoid prettier formatting like this? It'd be nice to have it returning `${propName}_rel {${type} {${nodeProps}} ${relationshipProps}}` all on one line (if it fits) as it's more readable